content/applications/finance/accounting/receivables/customer_invoices/electronic_invoicing.rst

```diff
@@ -0,0 +1,68 @@
+================================================================
+Electronic invoicing (:abbr:`EDI (electronic data interchange)`)
+================================================================
+
+EDI, or electronic data interchange, is the inter-company communication of business documents, such
+as purchase orders and invoices, in a standard format. Sending documents according to an EDI
+standard ensures that the machine receiving the message can interpret the information correctly.
+Various EDI file formats exist.
+
+EDI feature enables automating the administration between companies and might also be required by
+some governments for fiscal control or to facilitate the administration.
+
+Any business that processes large volumes of documents and forms can benefit from an EDI solution to
+increase automation and improve accuracy.
+
+Since July 2022, Odoo supports, among others, the following formats.
+
+.. list-table::
+   :header-rows: 0
+
+   * - Factur-X (CII)
+     - Default format on Odoo (enabled by default)
+   * - Peppol BIS Billing 3.0
+     - For companies which countries are part of the `EAS list
+       <https://docs.peppol.eu/poacc/billing/3.0/codelist/eas/>`_
+   * - E-FFF
+     - For Belgian companies
+   * - NLCIUS
+     - For Dutch companies
+   * - XRechnung UBL
+     - For German companies
+   * - EHF 3.0
+     - Norwegian companies have to use Peppol BIS Billing 3.0 that comply with the EHF 3.0 requirements
+
+.. seealso::
+   - :doc:`../../fiscal_localizations/overview/fiscal_localization_packages`
+   - :doc:`../../fiscal_localizations/overview/localizations_list`
+   - `Odoo and EDI <https://www.odoo.com/blog/odoo-news-5/odoo-and-edi-766#blog_content>`_
+
+Configuration
+=============
+
+Go to :menuselection:`Accounting --> Configuration --> Journals --> Customer Invoices -->
+Advanced Settings --> Electronic Invoicing` and enable the formats you need for this journal.
+
+.. image:: electronic_invoicing/formats.png
+   :align: center
+   :alt: Select the EDI format you need
+
+Once an electronic invoicing format is enabled, XML documents are generated, either visible or
+embedded in the PDF.
```

Comment:
FYI (maybe it can be useful to mention it)

the EDI documents are generated when clicking on the 'Confirm' button on top of the invoices/credit notes. 

The PDF are generated when clicking 'Send & Print' or 'Print' buttons. As E-FFF requires the PDF to be embedded in the xml (yes, yes, the PDF is inside the xml), it will only appear in the attachments when clicking on the 'Print' or 'Send & Print' button (after having confirmed the invoice).